LanguageFeatures/Augmentation-libraries/augmenting_types_A07_t02_lib.dart

```diff
@@ -10,15 +10,15 @@
 /// after this appending process, so you cannot have multiple `extends` on a
 /// class, or an `on` clause on an enum, etc.
 ///
-/// @description Checks that a class, mixin and enum augment may specify an
-/// additional `implements` clause
+/// @description Checks that a class, mixin, enum and extension type
+/// augmentations may specify additional `implements` clauses.
 /// @author sgrekhov22@gmail.com
 
 // SharedOptions=--enable-experiment=macros
 
 augment library 'augmenting_types_A07_t02.dart';
 
-interface class I2 {
+abstract interface class I2 implements I0 {
```

Comment:
Typo?:
```suggestion
augment interface class I2 implements I0 {
```

Reply:
Yes, typo, of course. Thank you for noticing!

Reply:
Thanks!